.github/generate-auto-update-template

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This is practically a workflow.yml document. It just has a section for expanding its arguments
+# into a YAML array.
+
+cat <<'EOT'
+#
+# This workflow was created automatically from the `auto-update-template.yml` by running `make -C .github workflows`
+# DO NOT EDIT THIS WORKFLOW
+#
+
+name: "auto-update-packages"
+on:
+  schedule:
+    # Update packages nightly
+    - cron:  '0 0 * * *'
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package-name:
+EOT
+
+for pkg in "$@"; do
+  if [[ $(make -sC ../vendor/$pkg info/auto-update-enabled) != "false" ]]; then
+    printf "        - %s\n" $pkg
+  else
+    echo Auto-update of $pkg is disabled >&2
+  fi
+done
```

Comment:
I much rather keep the logic out of here and in the pipeline. We already have the pipelines check if they are enabled (by checking `steps.info.outputs.package_enabled == 'true'`).  Now we have a different way here.